README.md

```diff
@@ -1,57 +1,44 @@
-# Project Name
+# Azure Data Tables sample application
 
-(short, 1-3 sentenced, description of the project)
+This is the sample project for using tha [Azure Tables SDK](https://www.npmjs.com/package/@azure/data-tables) library with the Azure Cosmos DB Table API.  The quickstart for this application is located at [https://docs.microsoft.com/azure/cosmos-db/table/create-table-nodejs](https://docs.microsoft.com/azure/cosmos-db/table/create-table-nodejs).
 
 ## Features
 
-This project framework provides the following features:
+This project demonstrates how to access the Cosmos DB Table API from a nodejs application using the Azure.Data.Tables package in the Azure SDK.  The Cosmos DB Table API is a schemaless NoSQL data store used to store tabular data.  Each row in an Azure Cosmos DB table has a partition key and row key which together form a unique key for the row.  All other properties (columns) of the row are dynamically added when objects are stored in the table.  In this way, there is no need to specify the properties (columns) of a table ahead of time as they are automatically added as needed.  If an object is loaded to the table that has new properties, the corresponding properties on the table will created automatically.
 
-* Feature 1
-* Feature 2
-* ...
+This sample application is based around storing weather data.  A base set of properties (temperature, humidity, pressure, wind direction, and wind speed) are stored for each observation loaded to a table.  However, the GUI can be used to create and store weather observations with additional properties (cloud cover, UV index, air quality index, etc) to demonstrate how the Table API automatically adds properties to a table as needed.  The ability to automatically extend a table in this way is useful for any application where the data sent to the application may dynamically change at any time.
 
 ## Getting Started
 
 ### Prerequisites
 
-(ideally very short, if any)
-
-- OS
-- Library version
-- ...
-
-### Installation
-
-(ideally very short)
-
-- npm install [package name]
-- mvn install
-- ...
+- An [Azure account](https://docs.microsoft.com/dotnet/azure/create-azure-account)
+- [LTS versions of Node.js](https://nodejs.org/en/)
```

Comment:
It is usually recommended that links don't include locale. I'd remove /en here and do a quick search to make sure there are no other links with locales